src/server/next-compat.test.ts

```diff
@@ -0,0 +1,114 @@
+import { NextRequest, NextResponse } from "next/server.js";
+import { describe, expect, it } from "vitest";
+
+import { toNextRequest, toNextResponse } from "./next-compat.js";
+
+describe("next-compact", () => {
```

Comment:
rename "next-compact" to "next-compat"
```suggestion
describe("next-compat", () => {
```